liblll/CodeFragment.cpp

```diff
@@ -523,14 +523,24 @@ void CodeFragment::constructOperation(sp::utree const& _t, CompilerState& _s)
 			requireSize(1);
 			requireDeposit(0, 1);
 
-			m_asm.append(Instruction::MSIZE);
```

Comment:
Please add a comment here explaining the rules:
- returns msize prior to expansion
- doesn't allocate in case of N = 0
- rounds N to a multiple of 32
- uses MLOAD to expand, which keeps memory untouched